frigate/http.py

```diff
@@ -81,6 +85,10 @@ def _db_close(exc):
 
     app.register_blueprint(bp)
 
+    app.wsgi_app = DispatcherMiddleware(
```

Comment:
I think there may be a better name than wsgi_app for this since right now it is Prometheus specific?

Reply:
Was following instructions here https://github.com/prometheus/client_python#flask

I believe it comes from from prometheus_client import make_wsgi_app

Reply:
Right, but that's an example. I believe the variable name can be whatever we want as the function and variable name shouldn't be dependent.

Reply:
Ah ok, when I get home from work i'll try this. Any name suggestions?

Reply:
Maybe `app.prometheus_proxy`

Reply:
I cant remove the wsgi_app part because its part of Flask

![image](https://user-images.githubusercontent.com/6330506/221705653-6291bf7d-2ed8-4e63-a221-286f1d4f849f.png)


if I do something like what you suggest

![image](https://user-images.githubusercontent.com/6330506/221706138-a029f124-7340-45b7-b798-0602ff76fd06.png)

I get 

![image](https://user-images.githubusercontent.com/6330506/221706223-5581cac6-09e5-47e5-aca9-6307ba6fe10a.png)


thoughts?

Reply:
Interesting, not what I expected. In any case thanks for trying it.